06-func.md

```diff
@@ -137,11 +136,9 @@ The computer does this for historical reasons:
 integer operations were much faster on early machines,
 and this behavior is actually useful in a lot of situations.
 It's still confusing,
-though,
-so Python 3 produces a floating-point answer when dividing integers if it needs to.
+therefore, Python 3 produces a floating-point answer when dividing integers if it needs to.
```

Comment:
I like dropping the 'though', but I don't think therefore makes grammatical sense here because "therefore" doesn't act as a conjunction like and, or, but, so; if you wanted to say therefore, you would need to start a new sentence. I'd suggest keeping 'so', but dropping 'thought'; i.e., `It's still confusing, so Python 3 produces...`